### ISSUE TITLE
Bowtie2 Version

### ISSUE THREAD

**Issue by BobLiterman** [rachelss/SISRS#66]
Hello all, 

As @anderspitman found earlier, different versions of Bowite2 give slightly different mapping data. I noticed that in the Docker version is using v.2.3.3.1, while I was using 2.3.4 locally, and 2.2.4 on my cluster. 

I think that it would be wise for everyone to use the same Bowtie2 version for testing, and in that vein I recommend that we all (and the Docker) upgrade to the latest major release (v.2.3.4). 

I ran a test while 2.3.3.1 and 2.3.4 give identical results, versions < 2.3 do not.

**Comment by anderspitman** [comment]
Agree. I forgot that this one of the biggest reasons I recommend Docker for
development testing. It's the easiest way to ensure a consistent environment
